app/controllers/expenses_controller.rb

```diff
@@ -22,7 +24,12 @@ def create
 
   private
 
+  def setup
```

Comment:
If this is becoming a pattern and they are needed in multiple places, always together, then probably they should be coupled even more, maybe with a view model. Not that you need to do it right now, but have that in mind.